IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml

```diff
@@ -2600,9 +2612,20 @@ components:
           $ref: '#/components/schemas/GeoCoordinate'
       required:
         - country
+        - stationCode
+        
+    StationCode:
+      type: object
+      properties:
+        codeList:
+          $ref: '#/components/schemas/StationCodeListDef'
+        code:
+          type: string
+          description: the code representing the station in the specified code-list     
+      required:
         - code
-      additionalProperties: false
```

Comment:
should we realy remove additional propüerties false? The resrtictions wuld then be in the text part of the specification only. To allow proprietary bilateral additions we could add a pattern restriction:

			"patternProperties": {
				"^p_": {
					"anyOf": [
						{
							"type": "object"
						},
						{
							"type": "string"
						}
					]
				}
			},

Reply:
Adressed

Reply:
I don't actually have a strong opinion here. OK for me to leave them in.